pkg/apis/serving/v1alpha2/util.go

```diff
@@ -0,0 +1,87 @@
+package v1alpha2
```

Comment:
Should we call this file "validation_util.go"

Reply:
In fact it's just a common place for code shared between explainer/transformer/predictor, and I moved some existing functions here too. Like the setResourceRequirementDefaults which is not for validation. And later more common code may come to this file so maybe just keep the name util.go?